src/io/flutter/preview/PreviewView.java

```diff
@@ -84,35 +81,30 @@
   @NotNull
   private final FlutterDartAnalysisServer flutterAnalysisServer;
 
-  final QuickAssistAction actionCenter;
-  final QuickAssistAction actionPadding;
-  final QuickAssistAction actionColumn;
-  final QuickAssistAction actionRow;
-  final QuickAssistAction actionContainer;
-  final QuickAssistAction actionMoveUp;
-  final QuickAssistAction actionMoveDown;
-  final QuickAssistAction actionRemove;
-  final ExtractMethodAction actionExtractMethod;
-  final ExtractWidgetAction actionExtractWidget;
+  private final InspectorGroupManagerService inspectorGroupManagerService;
+  private final InspectorGroupManagerService.Client inspectorStateServiceClient;
 
   private SimpleToolWindowPanel windowPanel;
-  private ActionToolbar windowToolbar;
-
-  private final Map<String, AnAction> messageToActionMap = new HashMap<>();
-  private final Map<AnAction, SourceChange> actionToChangeMap = new HashMap<>();
 
+  private boolean isSettingSplitterProportion = false;
+  private Splitter splitter;
+  private Splitter propertyEditSplitter;
   private JScrollPane scrollPane;
+  private JScrollPane propertyScrollPane;
   private OutlineTree tree;
+  private PreviewArea previewArea;
 
   private final Set<FlutterOutline> outlinesWithWidgets = Sets.newHashSet();
   private final Map<FlutterOutline, DefaultMutableTreeNode> outlineToNodeMap = Maps.newHashMap();
 
-  private VirtualFile currentFile;
+  private final EventStream<VirtualFile> currentFile;
```

Comment:
state that needs to be shared with subviews is moved into EventStream classes for easy listening.